src/lib.rs

```diff
@@ -14,10 +14,12 @@
 // You should have received a copy of the GNU General Public License
 // along with substrate-subxt.  If not, see <http://www.gnu.org/licenses/>.
 
-use crate::error::Result;
 use futures::future::Future;
 use jsonrpc_core_client::transports::ws;
-use parity_codec::Codec;
+use parity_codec::{
```

Comment:
`parity-codec` crate has been superseeded by https://crates.io/crates/parity-scale-codec.

Reply:
Can change this in a separate PR